README.md

```diff
@@ -2,9 +2,11 @@ TODO document all stages
 
 # Install tools
 
-On ubuntu linux, try running INSTALL.sh
+On ubuntu linux, try running `./INSTALL/ALL.sh`.
```

Comment:
‎This is just a test of commenting on diff lines in a pull request.

Reply:
Test reply 🙃